google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -381,6 +471,103 @@ private void registerOpenTelemetryMetrics() {
             .counterBuilder("append_rows")
             .setDescription("Counts number of incoming request rows")
             .build();
+    instrumentSentRequestCount =
+        writeMeter
+            .counterBuilder("append_requests_sent")
+            .setDescription("Counts number of requests sent over the network")
+            .build();
+    instrumentSentRequestSize =
+        writeMeter
+            .counterBuilder("append_request_bytes_sent")
+            .setDescription("Counts byte size of requests sent over the network")
+            .build();
+    instrumentSentRequestRows =
+        writeMeter
+            .counterBuilder("append_rows_sent")
```

Comment:
I think we should maintain less metrics. Can we just add a "result" field to append_requests/rows/bytes?

Reply:
Done. I have removed the following metrics: append_requests, append_request_bytes, append_rows, waiting_queue_length, connection_retry_count, append_requests_error, append_request_bytes_error, append_rows_error.

I now use the "error_code" attribute on each of the following metrics: append_requests_acked, append_request_bytes_acked, append_rows_acked, connection_end_count.